cloud/blockstore/tests/infra-cms/test.py

```diff
@@ -87,7 +87,7 @@ def remove_device(self, host, path):
         return r
 
 
-class TestCmsRemoveAgentNoUserDisks:
+class _TestCmsRemoveAgentNoUserDisks:
```

Comment:
Префикс _ - чтобы pytest не ругался в лог:
```
PytestCollectionWarning: cannot collect test class 'TestCmsRemoveAgentNoUserDisks' because it has a __init__ constructor (from: test.py)
```